opre_ops/ops_site/migrations/0002_auto_20210901_2327.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.6 on 2021-09-01 23:27
+
+import os
+
+from opre_ops.settings.local import BASE_DIR
+from django.db import migrations
+from csv import DictReader
+from ops_site.models import Agency
+from pathlib import Path
+
+def seed_agencies(apps, schema_editor):
+    filename = os.path.join('ops_site', 'migrations', 'seed_data', 'agencies.csv')
```

Comment:
_[caveat: I know this is a draft PR, you may have been planning to change this anyway!]_

Not such a big deal for this PR, but for future PRs in particular, I'd suggest using file names like `dummy-agency-data.csv`, `fake-invoice-amounts.csv`, etc just to make it super clear that this is dummy/fake data.

Reply:
Changed 😄

Reply:
Woohoo, thanks! I think this establishes a nice pattern!